apps/web/app/tailwind.css

```diff
@@ -0,0 +1,12 @@
+@tailwind base;
```

Comment:
Saw your comment on doing this another time, but for removing code, suggest we remove this in this PR, so it never enters the history